client/my-sites/current-site/style.scss

```diff
@@ -61,46 +61,33 @@
 .current-site__switch-sites {
 	background: lighten( $gray, 30% );
 	border-bottom: 1px solid lighten( $gray, 25% );
-	font-size: 13px;
 	display: block;
-	padding: 8px 8px 8px 40px;
-	color: darken( $gray, 20% );
+	padding: 8px;
 	box-sizing: border-box;
 	cursor: pointer;
 	position: relative;
 
+	.button {
+		text-align: left;
+		width: 100%;
+
+		.gridicon {
+			margin-right: 4px;
+		}
+	}
+
 	@include breakpoint( "<660px" ) {
 		background-color: $gray-light;
-		padding: 18px 16px 18px 40px;
 	}
 
 	&:hover {
 		background-color: $gray-light;
 		color: $blue-medium;
 
-		.current-site__site-count {
-			border-color: $white;
-			color: $white;
+		.button {
+			color: $gray-dark;
 		}
 	}
-
-	.gridicon {
-		position: absolute;
-			top: 9px;
-			left: 16px;
-
-		@include breakpoint( "<660px" ) {
-				top: 15px;
-			height: 24px;
-			width: 24px;
-		}
-	}
-}
-
-.rtl .current-site__switch-sites {
```

Comment:
Just to clarify, is this the intended RTL appearance? 

![RTL](https://cloud.githubusercontent.com/assets/1779930/12253146/a58bb672-b8ab-11e5-8799-8ec98adebc35.png)